README.md

```diff
@@ -280,11 +280,11 @@ describe('with a custom ruleset', () => {
           return acc.concat(pass.nodes);
         }, [])
         .reduce((acc, node) => {
-          return acc.concat(node.target);
+          return acc.concat(node.target.flat(1));
```

Comment:
was this intentional?

Reply:
Yes. Shadow dom us a double nested array `target` so trying to ask if the `flatPassesResults` includes an array would fail. So this just removes the nesting and keeps it a true flat array.